nexus/src/app/background/sync_switch_configuration.rs

```diff
@@ -801,7 +809,9 @@ impl BackgroundTask for SwitchPortSettingsManager {
                             rack_subnet: subnet,
                             infra_ip_first,
                             infra_ip_last,
+                            // FIX THIS
```

Comment:
What is this and the one below in reference to? The things to fix in this PR?

Reply:
Yes, that's what they were for. I was getting really tired and wanted to make sure I had visual indicators for myself (and others) of which parameters were being corrupted / malformed by the old logic.